app/femr/ui/controllers/HistoryController.java

```diff
@@ -102,7 +102,13 @@ public Result indexEncounterGet(int encounterId) {
         IndexEncounterMedicalViewModel indexEncounterMedicalViewModel = new IndexEncounterMedicalViewModel();
         IndexEncounterPharmacyViewModel indexEncounterPharmacyViewModel = new IndexEncounterPharmacyViewModel();
 
+        /* Alaa Serhan */
+        ServiceResponse<SettingItem> response = searchService.retrieveSystemSettings();
```

Comment:
Do we need to retrieve the settings here if all the patient/vital info is being converted in the service layer?